src/main/java/reposense/parser/StandaloneConfigJsonParser.java

```diff
@@ -27,6 +27,13 @@ public Type getType() {
      */
     @Override
     public StandaloneConfig parse(Path path) throws IOException {
-        return fromJson(path);
+        StandaloneConfig config = fromJson(path);
+
+        // Checks that config is properly parsed
+        if (config != null) {
+            config.initialize();
```

Comment:
whats the benefit of using `initialize();` over performing the null check in the getters itself?
The latter seems to be more resilience to me because dev/reviewer may missed out adding the field into the `initialize()` method.

Reply:
Good point. Since `StandaloneConfig` is a single use object, we can do null checks in the getters for simplicity.